client/me/purchases/credit-cards/credit-card-delete.tsx

```diff
@@ -23,11 +23,6 @@ import StoredCard from 'calypso/components/credit-card/stored-card';
 import PaymentMethodDeleteDialog from './payment-method-delete-dialog';
 import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
 
-/**
```

Comment:
Just above this line there's an import of `StoredCard` from `calypso/components/credit-card/stored-card`, but I think you've moved that file in this PR, so you'll need to update the imports.

Reply:
Oops. I may have been looking at a single commit and not the whole diff! Ignore me!